views/extras.tmpl

```diff
@@ -278,6 +278,10 @@
                 $("#target_percent").html(parseFloat(ex.pool_info.percent_target).toFixed(2))
                 $("#pool_size_percentage").html(parseFloat(ex.pool_info.percent).toFixed(2))
             }
+            //handling statuspage for future block 
+            if($("#futureblock").length ){
+                location.reload(true);
```

Comment:
please try to reload/refresh with turbolinks so as not to break the open websocket connection
 https://github.com/turbolinks/turbolinks/issues/329#issuecomment-341695200